src/scf-release/jobs/acceptance-tests/spec

```diff
@@ -0,0 +1,130 @@
+  acceptance_tests.nodes:
+    default: 2
+    description: The number of parallel test executors to spawn. The larger the number the higher the stress on the system.
+  acceptance_tests.verbose:
+    default: false
+    description: Whether to pass the -v flag to cf-acceptance-tests
+  acceptance_tests.include_apps:
+    description: Flag to include the apps test suite.
+  acceptance_tests.include_sso:
+    description: Flag to include the services tests that integrate with SSO.
+  acceptance_tests.include_security_groups:
+    description: Flag to include the security groups test suite.
+  acceptance_tests.include_internet_dependent:
+    description: Flag to include the internet dependent test suite.
+  acceptance_tests.include_services:
+    description: Flag to include the services API test suite.
+  acceptance_tests.include_v3:
+    description: Flag to include the v3 API test suite.
+  acceptance_tests.include_tasks:
+    description: Flag to include the v3 task tests dependent on the CC task_creation feature flag.
+  acceptance_tests.include_route_services:
+    description: Flag to include the route services tests. Diego must be deployed for these tests to pass.
+  acceptance_tests.include_routing:
+    description: Flag to include the routing tests. Diego must be deployed for these tests to pass.
+  acceptance_tests.include_diego_docker:
+    description: Flag to include tests related to running Docker apps on Diego. Diego must be deployed and the CC API docker_diego feature flag must be enabled for these tests to pass.
+  acceptance_tests.include_diego_ssh:
+    description: Flag to include the diego ssh suite
+  acceptance_tests.include_zipkin:
+    description: Flag to include the zipkin test suite. include_routing must be set as well for these tests to run. For the tests to pass router.tracing.enable_zipkin must also be set to true.
+  acceptance_tests.backend:
+    description: App tests push their apps using the backend specified. If not set, tests that require a specific backend will not be run. Route service tests require this flag to be set to diego in order to run.
+  acceptance_tests.include_backend_compatibility:
+    description: Flag to inlude whether we check DEA/Diego interoperability
+  acceptance_tests.include_detect:
+    description: Flag to run tests in the detect suite
+  acceptance_tests.use_http:
+    description: Flag for using HTTP when making api and application requests rather than the default HTTPS
+  acceptance_tests.include_privileged_container_support:
+    description: Flag for running tests that depend on privileged container support being enabled in your Cloud Foundry.
+  acceptance_tests.include_container_networking:
+    description: Flag for running tests that depend on container networking support being enabled in your Cloud Foundry.
+  acceptance_tests.include_isolation_segments:
+    description: Flag for running tests that depend on having isolation segments in your Cloud Foundry.
```

Comment:
Huh, I don't think we've been keeping up on the `include` flags. We're missing [`include_routing_isolation_segments`](https://github.com/cloudfoundry/cf-acceptance-tests/blob/cf1.21/helpers/config/config_struct.go#L94), for example.

Not in this PR, though; made [Trello card](https://trello.com/c/VQNQMVnl/) instead.